pkg/apis/config/defaults.go

```diff
@@ -68,6 +69,9 @@ func defaultConfig() *Defaults {
 func NewDefaultsConfigFromMap(data map[string]string) (*Defaults, error) {
 	nc := defaultConfig()
 
+	// Process bool field.
+	nc.EnableMultiContainer = strings.EqualFold(data["enable-multi-container"], "true")
```

Comment:
nit: If we ever change the default for this property to `true` this will override it to false if the property is not set in a defaults config

not important now

Reply:
I think that's WAI?  Alpha=Opt-In, Beta=Opt-Out, GA=remove the option.

Reply:
> Alpha=Opt-In, Beta=Opt-Out, GA=remove the option.

Yeah but `data["enable-multi-container"]` will return an empty string - so there's an explicit change that'll be needed when we swap from alpha to beta

not something that's necessary right now

Reply:
Yeah for beta this will require explicit code change, but I think that's by design, no? :)

Reply:
yeah  - that's why I just marked it a `nit`